main.cpp

```diff
@@ -168,15 +207,38 @@ struct DAW
         void convertMidi();
     };
     
-    //master music
-    void masterMusic(bool masteredMusic = true, bool error = false);
-    //process audio
+    
+    void masterMusic(bool masteredMusic = true);
     int processAudio(int NumberOfTracks = 24);
-    //mix music
     float mixMusic(float audioLevel = 0.9f);
 
 };
 
+void DAW::masterMusic(bool masteredMusic)
+{
+    if (numberofInputs == 2)
+    {
+     masteredMusic = true;
+    }
+}
+
+int DAW::processAudio(int NumberOfTracks)
+{
+    {
+    NumberOfTracks = 60;
+    }
+    return 0;
+}
+
+
+float DAW::mixMusic(float audioLevel)
+{
+    {
+    audioLevel = 99.9f;
+    }
```

Comment:
- You don't need these curly braces
- don't overwrite your function parameters!  use std::cout statements if you need to prevent 'variable unused' warnings.
- [ ] revise